packages/muelu/src/Misc/MueLu_ThresholdAFilterFactory_def.hpp

```diff
@@ -81,8 +81,9 @@ namespace MueLu {
     RCP<OMatrix> Ain = Get< RCP<OMatrix> >(currentLevel, varName_);
 
     // create new empty Matrix
-    RCP<CrsOMatrix> Aout = rcp(new CrsOMatrix(Ain->getRowMap(),Ain->getGlobalMaxNumRowEntries(),Xpetra::StaticProfile)); //FIXME
-
+    RCP<CrsOMatrix> Aout = rcp(new CrsOMatrix(Ain->getRowMap(),Ain->getGlobalMaxNumRowEntries() ? expectedNNZperRow_ <= 0 : expectedNNZperRow_ ,Xpetra::DynamicProfile)); //FIXME
```

Comment:
This looks backwards to me.  Isn't the condition being tested `expectedNNZperRow_ <= 0`?

Reply:
Oh dear, that's what happens with too much python..